osquery/dispatcher/scheduler.cpp

```diff
@@ -79,10 +80,12 @@ SQLInternal monitor(const std::string& name, const ScheduledQuery& query) {
     // Calculate a size as the expected byte output of results.
     // This does not dedup result differentials and is not aware of snapshots.
     size_t size = 0;
-    for (const auto& row : sql.rows()) {
+    for (const auto& row : sql.rowsTyped()) {
       for (const auto& column : row) {
         size += column.first.size();
-        size += column.second.size();
+        // Reviewers, does this seem reasonable, or should we do something like
+        // https://stackoverflow.com/questions/27012488/c-extract-size-of-boostvariant-element
+        size += sizeof(column.second);
```

Comment:
To keep backwards compatibility you would need to serialize each element and calculate the string size which makes no sense. What this column was measuring was anyway not the size of the output since that can only be calculated after serialization, and I'm not sure about the value of what this is calculating right now. I see two options here, (1) kill this column (2) move this calculation to after the query results are serialized (basically make `logQueryLogItem()` and `logQuerySnapshot()` return you the size of the serialized output and move the `recordQueryPerformance()` to the end). I would prefer option 2. Independently of the option to follow this should be submitted as a separate PR.

Reply:
Actually after thinking a bit more about this I would just kill this column as it doesn't make much sense to me, and it's going to require a bit a work.